### ISSUE TITLE
More instructions

### ISSUE THREAD

**Issue by ihavenokia** [ustwo/ustwo.com-frontend#573]
I wanted to try this project on my local machine, but I ca't figure out what I need to do. Will there be a step-by-step guide to make this run? Thanks

**Comment by jeanrauwers** [comment]
Have you checked the read me? I found lots of info there. it may help you out!

**Comment by ihavenokia** [comment]
@jeanrauwers yes, but I'm too noob to understand. It might be intuitive for those who work with those technologies, but as a .net developer it's hard to just dive in. There are so many concepts together and I can't find any good course to wrap them all
